accounts/views.py

```diff
@@ -11,7 +11,6 @@
 from django.utils.encoding import force_bytes, force_text
 from django.utils.html import strip_tags
 from django.utils.http import urlsafe_base64_encode, urlsafe_base64_decode
-
```

Comment:
Revert these to their previous state (you've deleted some newlines in these files).